examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -176,6 +205,127 @@ std::unique_ptr<GeneratedCluster> CreateCluster(chip::ClusterId id)
     return nullptr;
 }
 
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const Span<const char> & v)
+{
+    return wr.PutString(chip::TLV::Tag(), v);
+}
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const bool & v)
+{
+    return wr.PutBoolean(chip::TLV::Tag(), v);
+}
+
+template <typename T>
+CHIP_ERROR ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const T & v)
+{
+    return wr.Put(chip::TLV::Tag(), v);
+}
+
+CHIP_ERROR WriteValueToBuffer(const bool & value, uint8_t * buffer, const uint16_t & maxReadLength)
+{
+    if (maxReadLength != 1)
+    {
+        return CHIP_ERROR_BUFFER_TOO_SMALL;
```

Comment:
is `TOO SMALL` really true? it is if 0, but if 100, then it is too large? INVALID_ARGUMENT maybe? Other alternatives?

Reply:
Changed the condition to `== 0` here. No issue when the buffer size is larger than the value size. See [1872a7c](https://github.com/project-chip/connectedhomeip/pull/23840/commits/1872a7cf5840af04abe404e053917b6f58cf0f1f).